project/infra/code-index/src/cli.py

```
@@ -0,0 +1,260 @@
+        if len(response.errors) > 10:
+            console.print(f"  ... and {len(response.errors) - 10} more")
+
+
+@app.command()
+def search(
+    query: str = typer.Argument(..., help="Search query"),
+    top_k: int = typer.Option(10, "--top-k", "-k", help="Number of results"),
+    language: Optional[str] = typer.Option(
+        None, "--language", "-l", help="Filter by language"
+    ),
+    node_type: Optional[str] = typer.Option(
```

Ruff (0.15.2)

[warning] Unused function argument: `node_type`

(ARG001)